lib/mix/tasks/dogma.ex

```diff
@@ -11,8 +11,11 @@ defmodule Mix.Tasks.Dogma do
     {dir, formatter} = argv |> parse_args
     config = Config.build
 
+    {:ok, events} = GenEvent.start_link([])
```

Comment:
It could be in a supervision tree. If we're going to do that then we should give it a global name, since the PID will change when it restarts.

I'm not sure how to go about re-adding event handlers when it restarts, so it might just be best to let it fail for now and tackle that later. I guess it depends, if this crashes would it be better for the entire app to crash, or for the app to proceed and the only indication of ultimate success or failure be the exit code?